enabler/src/de/schildbach/pte/dto/Line.java

```diff
@@ -54,31 +55,58 @@
 
 	public Line(final String id, final String network, final Product product, final String label)
 	{
-		this(id, network, product, label, null, null, null);
+		this(id, network, product, label, null, null, null, null);
+	}
+
+	public Line(final String id, final String network, final Product product, final String label, final String name)
```

Comment:
I think we only need to add the 1 constructor you're using, and the new version with all parameters. Others can be added as needed.